torch/testing/_internal/common_methods_invocations.py

```diff
@@ -13655,29 +13656,60 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
                     supports_fwgrad_bwgrad=True,
                     assert_autodiffed=True,
                     supports_one_python_scalar=True,
-                    # TODO: FIXME: pow needs a way of specifying that for integer
-                    #   types only it does not support negative exponentes
-                    rhs_make_tensor_kwargs=dict(low=1),
+                    # Raising integer values to negative values is not causes an error
+                    rhs_make_tensor_kwargs=dict(low=0),
+                    # Logarithims of negative values are outside the domain of the function
```

Comment:
nit: logarithim -> logarithm

What do you mean by this comment? Do you mean to say something about raising negative numbers to fractional powers?